src-ble/linux/simpledbus/base/Logger.cpp

```diff
@@ -20,8 +20,7 @@ Logger* Logger::get() {
     return &instance;
 }
 
-void Logger::print_log(std::string message) {
-    std::scoped_lock lock(_mutex);
+void Logger::print_log_needslock(std::string message) {
```

Comment:
Good catch! That's what happens when one codes this in a hurry :P

We can keep the original function name, the idea for the future is to allow more flexibility in deciding where the logs actually go.

Reply:
name reverted.  thanks for your work making this library.  it was pretty hard to find; I bet there are other similar libraries hiding out there somewhere.